broker-daemon/proto/broker.proto

```diff
@@ -77,6 +77,8 @@ message HealthCheckResponse {
   repeated EngineStatus engine_status = 1;
   // The status of the relayer this BrokerDaemon is connected to, either `'OK'` or the error message returned when connecting to it.
   string relayer_status = 2;
+  // The status of every engine available on the BrokerDaemon.
+  repeated OrderbookStatus orderbook_status = 3;
```

Comment:
update the comments above the `rpc HealthCheck` to include the orderbook status (both js and shell outputs)